.travis.yml

```diff
@@ -24,17 +24,17 @@ before_install:
   - if [[ $TRAVIS_PYTHON_VERSION == 3* ]]; then 2to3 -nw pysal/ > /dev/null; fi
 
 install:
-  - conda install --yes numpy scipy nose pip
+  - conda install --yes pip
   - if [ "$PYSAL_PLUS" == true ]; then
-        echo 'plus testing'; conda install --yes --file requirements_plus.txt;
-        else conda install --yes --file requirements.txt; 
+        echo 'plus testing'; pip install .[plus];
+    else pip install .;
     fi;
-  - pip install -r travis.txt
+  - pip install .[dev]
 
```

Comment:
Yeah, I think this is why the original author went with the separate `travis.txt` testing dependency list, especially since it requires stuff like `nose-exclude`. Definitely why I kept it in this pattern in #834 

Reading the requirements for a target from file in the `setup.py` would be a fine implementation, too!

The critical part is the install for users needs to grab at least numpy and scipy. Development setup and CI setup should be documented, but doesn't necessarily have to work solely through a `pip` install (especially since the pysal-plus environment has a gdal dependency, so I'm not even sure that can work in any event). Not sure what the fellow maintainers think!